web/packages/teleport/src/Discover/Database/CreateDatabase/useCreateDatabase.ts

```diff
@@ -162,7 +163,7 @@ export function useCreateDatabase() {
       });
   }
 
-  function fetchDatabaseServers(query: string, limit: number) {
+  function fetchDatabaseServers(query: string, limit = 1000) {
```

Comment:
What's the configurable limit for if we never use it? I think it'd be warranted if some teleport.e component wanted to use this function with a different limit. But it seems to be called only from one place, so we might as well hardcode it below.

Reply:
i removed it altogether, backend defaults to 1k if limit is not defined